samples/minimal-backend/client/package.json

```
@@ -9,8 +9,8 @@
   "dependencies": {
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
-    "@docker/docker-mui-theme": "^0.0.4",
-    "@docker/extension-api-client": "^0.2.2",
+    "@docker/docker-mui-theme": "^0.0.8",
```

Should we change this to `<0.1.0` as we did in the `init` command?
```suggestion
    "@docker/docker-mui-theme": "<0.1.0",
```
same in other package.json files.